### DISCUSSION TITLE
How to update to the latest vueform version via the official vueform/nuxt module?

### DISCUSSION THREAD

**Discussion by alahanoun** [vueform/vueform#318] [Questions] [answered]
We are currently using vueform through the latest version of the official nuxt module, however the vueform version installed is 1.9.5, and we need help updating it to version 1.10.4.
Thank you!

**Comment by adamberecz** [comment, marked as answer]
An npm reinstall (`rm -R node_modules package-lock.json && npm install`) would probably solve the issue.

If not, you can run `npm install @vueform/vueform` in your project directory—that will ensure you will have the latest Vueform version.
